---a/net/minecraft/network/play/client/C0CPacketInput.java
+++b/net/minecraft/network/play/client/C0CPacketInput.java
@@ -1,32 +1,25 @@
 package net.minecraft.network.play.client;
 
 import java.io.IOException;
+
 import net.minecraft.network.INetHandler;
 import net.minecraft.network.Packet;
 import net.minecraft.network.PacketBuffer;
 import net.minecraft.network.play.INetHandlerPlayServer;
-import net.minecraftforge.fml.relauncher.Side;
-import net.minecraftforge.fml.relauncher.SideOnly;
 
-public class C0CPacketInput implements Packet
-{
+
+public class C0CPacketInput implements Packet {
+
     private float strafeSpeed;
     private float forwardSpeed;
     private boolean jumping;
     private boolean sneaking;
     private static final String __OBFID = "CL_00001367";
 
-    public C0CPacketInput() {}
-
-    @SideOnly(Side.CLIENT)
-    public C0CPacketInput(float strafeSpeed, float forwardSpeed, boolean jumping, boolean sneaking)
-    {
-        this.strafeSpeed = strafeSpeed;
-        this.forwardSpeed = forwardSpeed;
-        this.jumping = jumping;
-        this.sneaking = sneaking;
+    public C0CPacketInput() {
     }
 
-    public void readPacketData(PacketBuffer buf) throws IOException
-    {
+    public void readPacketData(PacketBuffer buf) throws IOException {
         this.strafeSpeed = buf.readFloat();
         this.forwardSpeed = buf.readFloat();
         byte b0 = buf.readByte();
@@ -36,53 +36,43 @@
         this.sneaking = (b0 & 2) > 0;
     }
 
-    public void writePacketData(PacketBuffer buf) throws IOException
-    {
+    public void writePacketData(PacketBuffer buf) throws IOException {
         buf.writeFloat(this.strafeSpeed);
         buf.writeFloat(this.forwardSpeed);
         byte b0 = 0;
 
-        if (this.jumping)
-        {
-            b0 = (byte)(b0 | 1);
+        if (this.jumping) {
+            b0 = (byte) (b0 | 1);
         }
 
-        if (this.sneaking)
-        {
-            b0 = (byte)(b0 | 2);
+        if (this.sneaking) {
+            b0 = (byte) (b0 | 2);
         }
 
         buf.writeByte(b0);
     }
 
-    public void processPacket(INetHandlerPlayServer handler)
-    {
+    public void processPacket(INetHandlerPlayServer handler) {
         handler.processInput(this);
     }
 
-    public float getStrafeSpeed()
-    {
+    public float getStrafeSpeed() {
         return this.strafeSpeed;
     }
 
-    public float getForwardSpeed()
-    {
+    public float getForwardSpeed() {
         return this.forwardSpeed;
     }
 
-    public boolean isJumping()
-    {
+    public boolean isJumping() {
         return this.jumping;
     }
 
-    public boolean isSneaking()
-    {
+    public boolean isSneaking() {
         return this.sneaking;
     }
 
-    public void processPacket(INetHandler handler)
-    {
-        this.processPacket((INetHandlerPlayServer)handler);
+    public void processPacket(INetHandler handler) {
+        this.processPacket((INetHandlerPlayServer) handler);
     }
-}
+}
